unison-cli/src/Unison/Codebase/Transcript/Runner.hs

```diff
@@ -300,16 +300,21 @@ run isTest verbosity dir stanzas codebase runtime sbRuntime nRuntime config ucmV
             _ <- liftIO (writeIORef mStanza maybeStanza)
             case maybeStanza of
               Nothing -> do
-                liftIO (putStrLn "")
+                liftIO (putStrLn "\r✔️   Completed transcript.                   ")
                 pure $ Right QuitI
-              Just (s, idx) -> do
+              Just (s, midx) -> do
                 unless (Verbosity.isSilent verbosity) . liftIO $ do
                   putStr $
-                    "\r⚙️   Processing stanza "
-                      ++ show idx
-                      ++ " of "
-                      ++ show (length stanzas)
-                      ++ "."
+                    maybe
+                      "\r⏩   Skipping non-executable Markdown block."
+                      ( \idx ->
+                          "\r⚙️   Processing stanza "
+                            ++ show idx
+                            ++ " of "
+                            ++ show (length stanzas)
+                            ++ ".          "
```

Comment:
😑
Could we pull out the text constants and use `strlen` and `max` and arithmetic instead of hard-coding the spaces? 😄
Otherwise looks good.

Reply:
Ah, it looks like we have [`Unison.PrettyTerminal.clearCurrentLine`](https://github.com/unisonweb/unison/blob/trunk/lib/unison-pretty-printer/src/Unison/PrettyTerminal.hs#L34), which should do it.

There’s also `^[[2K`, but I don’t know how we feel about VT100 escapes.

Reply:
https://github.com/unisonweb/unison/blob/trunk/lib/unison-pretty-printer/src/Unison/PrettyTerminal.hs#L34 sounds good for now; I wouldn't wanna print escapes directly without conditioning on some sort of terminal detection.